Gemfile.lock

```diff
@@ -95,9 +95,9 @@ GEM
     grape-entity (0.5.1)
       activesupport
       multi_json (>= 1.3.2)
-    grape-swagger (0.23.0)
+    grape-swagger (0.25.3)
```

Comment:
Ah, can you remove all the changes to this file as well? Thank you, StellaMaris!